cryptlex/lexfloatclient/lexfloatclient.py

```diff
@@ -117,7 +123,30 @@ def GetFloatingClientLibraryVersion():
         status = LexFloatClientNative.GetFloatingClientLibraryVersion(buffer,buffer_size)
         if status != LexFloatStatusCodes.LF_OK:
             raise LexFloatClientException(status)
-        return LexFloatClientNative.byte_to_string(buffer.value)   
+        return LexFloatClientNative.byte_to_string(buffer.value)  
+     
+    @staticmethod
+    def GetHostConfig():
+        """Gets the host configuration.
+
+        Raises:
```

Comment:
@muneebkq This line in docs is missing:

This function sends a network request to LexFloatServer to get the configuration details.